client/src/uxManager.js

```diff
@@ -49,7 +49,7 @@ class UxManager extends LuigiClientBase {
     helpers.sendPostMessageToLuigiCore({ msg: 'luigi.remove-backdrop' });
   }
   /**
-   * This method informs the main application that there are unsaved changes in the current view in the iframe. For example, that can be a view with form fields which were edited but not submitted.
+   * This method informs the main application that there are unsaved changes in the current view in the iframe. It can be used to prevent navigation away from the current view. For example, it can be used with form fields which were edited but not submitted, although the functionality is not restricted to forms.
```

Comment:
@alexandra-simeonova I was reading it the first time and I feel like the 2nd occurrence of the phrase "it can be used" can be removed since the context is clear enough. What do you think?